lesson07/assignment/README.md

```diff
@@ -8,6 +8,7 @@ In the last assignment in lesson06, you created a responsive website with Bootst
 - [Vue DevTools - Firefox](https://addons.mozilla.org/en-US/firefox/addon/vue-js-devtools/)
 - [Adding Bootstrap to Vue CLI](https://travishorn.com/adding-bootstrap-to-a-vue-cli-project-98c2a30e0ed0)
 - [Bootstrap Vue](https://bootstrap-vue.js.org/docs/)
+- Depending on which Bootstrap components you are using, you can choose to just use Bootstrap instead of Bootstrap Vue.  Bootstrap Vue components can be powerful and save you time if you need the functionality.  Choosing Bootstrap over Bootstrap Vue will prevent some extra refactoring that would be needed to get your code snippets from the assignment in lesson 06.
```

Comment:
Maybe move this information to the line above, as it's extra context about that tool?

Reply:
Good call.  Makes more sense to give clarification before linking to the tools.